File: WinForms/CS/SpreadsheetMainDemo/Modules/Charting.cs
using DevExpress.XtraBars.Ribbon;

namespace DevExpress.XtraSpreadsheet.Demos {
    public partial class ChartingModule : SpreadsheetRibbonTutorialControlBase {
        public ChartingModule() {
            InitializeComponent();
            spreadsheetControl1.Options.Culture = DefaultCulture;
            spreadsheetControl1.LoadDocument(DemoUtils.GetRelativePath("Charting.xlsx"));
        }

        protected internal override RibbonPage SelectedRibbonPage { get { return insertRibbonPage1; } }
    }
}

File: WinForms/VB/SpreadsheetMainDemo/Modules/Charting.vb
Imports DevExpress.XtraBars.Ribbon

Namespace DevExpress.XtraSpreadsheet.Demos

    Public Partial Class ChartingModule
        Inherits SpreadsheetRibbonTutorialControlBase

        Public Sub New()
            InitializeComponent()
            spreadsheetControl1.Options.Culture = DefaultCulture
            spreadsheetControl1.LoadDocument(DemoUtils.GetRelativePath("Charting.xlsx"))
        End Sub

        Protected Friend Overrides ReadOnly Property SelectedRibbonPage As RibbonPage
            Get
                Return insertRibbonPage1
            End Get
        End Property
    End Class
End Namespace
